cblearn/datasets/_linear_subspace.py

```diff
@@ -0,0 +1,185 @@
+from cblearn.datasets._base import BaseManifold
+import numpy as np
+from scipy.stats import ortho_group
+from sklearn.utils import check_random_state
+from typing import Union, Dict, Callable
+from scipy.spatial.distance import pdist, squareform
+
+
+class LinearSubspace(BaseManifold):
+    """
+    Linear Subspace
+
+    Linear Subspace is a class for creating a hyperplane of a given
+    subspace dimension embedded in a higher dimensional space. It gives
+    a method of generating synthetic ordinal data with intrinsic
```

Comment:
You write "generating synthetic ordinal data" In what sense is the generated data ordinal? Doesn't this class generate points?